java/access-token-client/src/main/java/no/bankaxept/epayment/client/accesstoken/AccessTokenRetriever.java

```diff
@@ -0,0 +1,25 @@
+package no.bankaxept.epayment.client.accesstoken;
+
+import no.bankaxept.epayment.client.base.accesstoken.AccessTokenPublisher;
+import no.bankaxept.epayment.client.base.accesstoken.AccessTokenSubscriber;
+import no.bankaxept.epayment.client.base.accesstoken.ScheduledAccessTokenPublisher;
+import no.bankaxept.epayment.client.webflux.WebFluxClient;
+
+import java.time.Duration;
+
+public class AccessTokenRetriever {
```

Comment:
I found it easier adding this token retriever like a module in this repo, so I could add the webflux-client with it

Reply:
I have moved it back to platform